thread_options.h

```diff
@@ -328,6 +329,7 @@ struct thread_options_pack {
 	uint8_t opendir[FIO_TOP_STR_MAX];
 	uint8_t ioengine[FIO_TOP_STR_MAX];
 	uint8_t mmapfile[FIO_TOP_STR_MAX];
+	uint16_t start_offset_percent;
```

Comment:
It might be easier to add this to the end of the struct and then you won't have to add the pads...